### ISSUE TITLE
IE11 support is broken again

### ISSUE THREAD

**Issue by scttdavs** [matthewp/haunted#168]
Looks like a change 5 months ago broke support for IE11, even with the proxy polyfill. The polyfill does not support `getPrototypeOf`:
https://github.com/GoogleChrome/proxy-polyfill/blob/f4e1a0651833b0e93e016c4e1a511171d973f9b4/src/proxy.js#L52

This PR is where it broke:
https://github.com/matthewp/haunted/pull/128

**Comment by lsimone** [comment]
we are having the same problem.
We are building a component library that should support IE11 too.

I like very much haunted, so I tried to make a PoC and build an hello world with `haunted` and [`lit-element`](https://lit-element.polymer-project.org/).

Even if I like the haunted approach with react-like hooks, I haven't been able to make it work on IE11, while `lit-element` is currently working with the build configuration I wrote in [the PoC project](https://github.com/lsimone/webcomponents-lib).

Do you see anything wrong with my code?
Could you please give us some hints about how to solve it, or it's just not feasible?

Thanks

**Comment by CKGrafico** [comment]
Same to me, not working in IE11

**Comment by matthewp** [comment]
I'd accept a PR that conditionally adds that proxy trap only in supported browsers.

**Comment by CKGrafico** [comment]
Yes sounds good, but can you confirm that this is a bug, and your readme is not ok? becuase maybe everything works to you and there is another problem.

**Comment by CKGrafico** [comment]
FYI: as a temp fix you can comment `getPrototypeOf`  from the lib and your project will work

**Comment by matthewp** [comment]
Not sure what you mean. This project has never officially supported IE11. Some contributors discovered a way to get it to work and they documented it via the readme. I'm willing to make small changes such as this one to get it to continue to work but if it comes to the point where we have to dramatically do things differently to support IE11 then we'll just drop support.

**Comment by CKGrafico** [comment]
For sure :) the only change right now is the `getPrototypeOf` you'll need to decide if is enough important or not
